lib/Grape.js

```diff
@@ -45,7 +45,8 @@ class Grape extends Events {
       host: this.conf.host || false,
       bootstrap: this.conf.dht_bootstrap,
       timeBucketOutdated: this.conf.dht_nodeLiveness,
-      verify: crypto.verify
+      verify: crypto.verify,
+      maxAge: this.conf.timeslot
```

Comment:
@mafintosh can we call this `dht_timeslot` or `dht_peer_maxAge` ?

Reply:
@mafintosh would be cool to support it as parameter here https://github.com/bitfinexcom/grenache-grape/blob/master/bin/grape.js